source/includes/administration/_service_connections.md

```diff
@@ -59,3 +59,92 @@ Attributes | &nbsp;
 `name`<br/>*string* | The name of the service connection
 `type`<br/>*string* | The type of the service connection.
 `status`<br/>*Object* | Status of the service connection. Tells you if the service is up.<br/>*includes*: `lastUpdated`, `reachable`
+
+<!-------------------- GET APIINFO -------------------->
+
+### Retrieve API credentials
+
+`GET /{serviceConnectionId}/apiInfo`
+
+```shell
+# Retrieve your API credentials
+curl -X GET \
+  "https://system.cloudmc-dev.cloudops-devteam.com/rest/services/connections/5b0172cf-00bd-43de-b2e8-9aaf217c8c35/apiInfo?environmentId=ce3b76e6-c4e1-4e11-850d-e31a3055dc8e" \
+  -H "MC-Api-Key: your_api_key" \
+# Response body example
+```
+```json
+{
+    "data": {
+        "canRegenerateCredentials": true,
+        "fields": [{
+            "key": "endpoint",
+            "value": "https://your_endpoint.com/auth",
+            "sensitive": false
+        }, {
+            "key": "api_key",
+            "value": "my_api_key",
+            "sensitive": true
+        }, {
+            "key": "secret_key",
+            "value": "my_secret_key",
+            "sensitive": true
+        }, {
+            "key": "project_id",
+            "value": "079bdead-61b5-4b38-86ed-dbbf963808ec",
+            "sensitive": false
+        }]
+    }
+}
+```
+
+Retrieve your API keys for some environment within a service. A user can only access their own API keys, and only for environments where they are a member.
+
+Query Parameters (*required*) | &nbsp;
+---------- | -----
+`environmentId`<br/>*UUID* | The id of the environment.
+
+
+Attributes | &nbsp;
+---- | -----------
+`canRegenerateCredentials`<br/>*boolean* | True if the user can regenerate their API keys for the service.
+`fields`<br/>*Array[object]* | An array of key-value pairs describing the users' API keys for the service.<br/>*includes*: `key`, `value` and `sensitive`
```

Comment:
An array of objects that describe different parameters _(endpoint, api_key, secret_key, project_id)_ comprised in the users' API credentials
*includes*: `key`, `value` and `sensitive`